runtime/service_integ_test.go

```diff
@@ -1364,6 +1364,34 @@ func TestStopVM_Isolated(t *testing.T) {
 				require.Equal(status.Code(err), codes.OK)
 			},
 		},
+		{
+			name: "Jailer SIGKILL",
+			createVMRequest: proto.CreateVMRequest{
+				JailerConfig: &proto.JailerConfig{
+					UID: 300000,
+					GID: 300000,
+				},
+			},
+			stopFunc: func(ctx context.Context, fcClient fccontrol.FirecrackerService, vmID string) {
+				firecrackerProcesses, err := internal.WaitForProcessToExist(ctx, time.Second, findFirecracker)
+				require.NoError(err, "failed waiting for expected firecracker process %q to come up", firecrackerProcessName)
+				require.Len(firecrackerProcesses, 1, "expected only one firecracker process to exist")
+				firecrackerProcess := firecrackerProcesses[0]
+
+				err = firecrackerProcess.KillWithContext(ctx)
+				require.NoError(err, "failed to kill firecracker process")
+
+				// Sleep here to ensure runc finishes execution
+				time.Sleep(100 * time.Millisecond)
+
+				// ensure that the jailer has cleaned up all of the containers
+				b, err := exec.Command("runc", "list").Output()
+				require.NoError(err, "failed to run 'runc list'")
+				lines := strings.Split(string(b), "\n")
+				expectedLines := 2
+				assert.Equal(expectedLines, len(lines), string(b))
```

Comment:
* You can use `runc list --quiet` to remove the header
* Shouldn't be 1 (only the header)?. Why do we have 2 lines?

Reply:
Naw, two. For example the string "foo\n", will split into ["foo", ""]

Reply:
Can we use `go-runc` to interact with runc (same applies for `Close` above) ? It provides high level interface to runc, does output parsing as needed as we already have it pulled in.

Reply:
Done